src/ERC1967Pausable.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.7;
+
+import {StorageSlot} from "openzeppelin-contracts/utils/StorageSlot.sol";
+import {ERC1967Ownable} from "./ERC1967Ownable.sol";
+
+/// @notice The proxy-safe pausability mix-in.
+/// Only the owner can pause or unpause the contract.
+/// It's supposed to be used only in contracts behind a proxy.
+///
+/// This contract uses a custom ERC-1967 slot to store its state.
+/// All instances of `ERC1967Pausable` contracts are paused and can't be unpaused.
+/// When a proxy uses such contract via delegation, it's initially unpaused.
+abstract contract ERC1967Pausable is ERC1967Ownable {
```

Comment:
What about removing `ERC1967Ownable` entirely. 

Make the methods virutal.
```

    /// @notice Triggers stopped state.
    function pause() public virtual whenNotPaused  {
        _setPaused(true);
        emit Paused(msg.sender);
    }

    /// @notice Returns to normal state.
    function unpause() public virtual whenPaused  {
        _setPaused(false);
        emit Unpaused(msg.sender);
    }
```

Than bring everything together in the ManagerDrip layer?

Reply:
Implemented in the #75

Reply:
Fixed